lib/internal/Magento/Framework/Config/Data/ConfigDataFactory.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\Config\Data;
+
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\ObjectManagerInterface;
+
+/**
+ * Factory for ConfigData
+ * @api
```

Comment:
So since it is a patch release we cannot add these tag currently. @renttek Can you check all the classes and remove the `@api` tag if you added it in.